src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -465,6 +445,15 @@ public SignatureValidator SignatureValidator
         }
 
         /// <summary>
+        /// Gets or sets the <see cref="ReadOnlyCollection{SecurityKey}"/> that is to be used for decrypting inbound tokens.
+        /// </summary>
+        public IList<SecurityToken> TokenDecryptionKeys
```

Comment:
Let's decide if this should be IList<>, ICollection<>, IEnumerable<>. Also return ...<SecurityKey>, instead of ...<SecurityToken>.